SYCL/AtomicRef/add.h

```diff
@@ -287,10 +287,6 @@ template <access::address_space space, typename T, typename Difference = T,
 void add_test_scopes(queue q, size_t N) {
   std::vector<memory_scope> scopes =
       q.get_device().get_info<info::device::atomic_memory_scope_capabilities>();
-  if (std::find(scopes.begin(), scopes.end(), memory_scope::system) !=
```

Comment:
why are these clauses removed?  If we find `memory_scope::system` shouldn't we test it? And, if we do not, then nothing will happen anyway, no?

Reply:
As explained in top level comment, these would now have to be in a separate file as they have a different sm version requirement. As `atomic_ref` tests are among the slowest and there is quite a number of them, this would significantly increase running time for test suite. Instead I opted to just remove those.